### ISSUE TITLE
Service Verification letter - Accessibility issues 

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16512]
## What Happened?
**VoiceOver / TalkBak**

1. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 with TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 1: 
1. With the appropriate screen reader enabled on iOS or Android, navigate to the service verification letter page
2. Select the back button
3. Note that a11y hint is not picked up by the screen reader 

## Desired Behavior
**VoiceOver / TalkBak**
     
- NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)  🔽

For 1:
- With the appropriate screen reader enabled on iOS or Android, the back button a11yHint will be picked up as "Navigates to the previous page" 

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #13942

**Comment by esther-k** [comment]
**12/21**

Pass - on iOS and Android. The back button a11yHint is now using the general a11yHint.
